wormhole-connect/src/hooks/useAmountValidation.ts

```diff
@@ -49,37 +49,30 @@ export const useAmountValidation = (props: Props): HookReturn => {
     [props.quotesMap],
   );
 
-  const allRoutesFailed = useMemo(
-    () =>
-      props.routes.every((route) => props.quotesMap[route]?.success === false),
-    [props.routes, props.quotesMap],
-  );
+  const allRoutesFailed = useMemo(() => {
+    if (Object.keys(props.quotesMap).length === 0) {
```

Comment:
this fixes a bug unrelated to `amount.Amount` refactor: if you run `every` on an empty array, it always returns `true`.